ttnn/cpp/ttnn/operations/eltwise/unary/common/unary_op_utils.cpp

```diff
@@ -300,6 +301,18 @@ std::pair<std::string, std::string> get_op_init_and_func_parameterized(
                 "unary_min_tile_init();",
                 fmt::format("unary_min_tile({}, {:#x}u);", idst, std::bit_cast<uint32_t>(param0))};
             break;
+        case UnaryOpType::SELU: {
+            TT_ASSERT(params.size() == 2, "Expected selu to take 2 parameters");
```

Comment:
Is there a reason we don't want this checked in release mode? (given that's typically the only build that the pipeline runs tests on)